modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc

```diff
@@ -276,7 +276,7 @@ By enabling etcd encryption for the key values in etcd, you will incur a perform
 <7> The output includes a custom command that you can run to create a cluster with the same configuration in the future.
 --
 +
-As an alternative to using the `--interactive` mode, you can specify the customization options directly when you run `rosa create cluster`. Run `rosa create cluster --help` to view a list of available CLI options.
+As an alternative to using the `--interactive` mode, you can specify the customization options directly when you run `rosa create cluster`. Run `rosa create cluster --help` to view a list of available CLI options, or refer to link:https://access.redhat.com/documentation/en-us/red_hat_openshift_service_on_aws/4/html/rosa_cli/rosa-managing-objects-cli#rosa-create-cluster-command_rosa-managing-objects-cli[Managing objects with the rosa CLI: create cluster].
```

Comment:
This needs an xref as well.

Reply:
Corrected, thanks!

Reply:
So, we are not allowed to use a link as a work around for the "no xrefs in modules" rule. The work around is to refer to the section of the docs without a link, so something like:
"For more information, see _Managing objects with the ROSA CLI_." Then, in the assembly, you use the `.Additional resources` label to create an additional resources section as a trailing include. In this section, you add the xref to the documentation. 
This is decribed in the repo guidelines in a couple of places:
* https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#writing-assemblies
* https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections

Reply:
OK, so these cannot be link or xref and should be a link-free reference instead. Thankyou!